test/spec/dataviews/category-dataview-model.spec.js

```diff
@@ -146,6 +146,34 @@ describe('dataviews/category-dataview-model', function () {
     expect(this.model._fetch.calls.count()).toEqual(1);
   });
 
+  describe('filters over data', function () {
```

Comment:
Since we're unit testing two different methods, I would change this to:

```
describe('numberOfAcceptedCategories, function () {
  ...
});

describe('numberOfRejectedCategories, function () {
  ...
});
```